app/views/layouts/main.volt

```diff
@@ -163,6 +163,11 @@
                                         Gab.ai
                                     </a>
                                 </li>
+                                <li>
+                                    <a href="https://phalcon.link/discord" target="_blank">
+                                        Discord
+                                    </a>
```

Comment:
Actually it is Community menu, eg. Social, not Chats. We have the Discord nearby with the Slack.